common/menu-list.c

```diff
@@ -189,6 +206,15 @@ int themeMenuScan(const char* target) {
 
     closedir(dir);
     menuSort();
+
+    menuEntry_s* me = menuCreateEntry(ENTRY_TYPE_THEME);
+    strncpy(me->path, "", sizeof(me->path)-1);
```

Comment:
Remove these two lines, menuCreateEntry already ends up clearing the entire entry.

Reply:
what two lines?

Reply:
the stncpy? and me->path last elem = 0?

Reply:
also... if i were to add a + and - button on the screen so users can see they can press + to load theme menu and - to load the netloader... where would you want them.... o.o

Reply:
"the stncpy? and me->path last elem = 0?" yes

" if i were to add" No, it can be done later.

Reply:
oops.... i already added the code for the minus and plus button...